scripts/exact_1d_bayesian_filtering_example3.py

```diff
@@ -0,0 +1,79 @@
+import functools
+
+from jax import numpy as jnp
+from jax import scipy as jsp
+from jax import random
+import matplotlib.pyplot as plt
+
+import exact_1d_bayesian_filtering_utils as utils
+
+
+# state transition function
+def f(x, v, k=None):
+    return 0.7 * x + v
+
+
+# measurement function
+def sat(x, minval, maxval):
+    return jnp.maximum(jnp.minimum(x, maxval), minval)
+
+
+def h(x, e):
+    return sat(x + e, minval=-1.5, maxval=1.5)
+
+
+# to get x from measurement without noise
+def inv_h(y):
+    return y
+
+
+# functions to get sample
+v_rvs = lambda rng_key, shape: random.normal(rng_key, shape=shape)
+e_rvs = lambda rng_key, shape: random.normal(rng_key, shape=shape) * jnp.sqrt(0.5)
+x0_rvs = lambda rng_key, shape: random.normal(rng_key, shape=shape) * jnp.sqrt(0.1)
```

Comment:
It's better to use `def` even if defining one-liner functions. See: https://peps.python.org/pep-0008/#programming-recommendations